airflow/providers/amazon/aws/transfers/gcs_to_s3.py

```diff
@@ -177,7 +182,9 @@ def execute(self, context):
                 dest_key = self.dest_s3_key + file
                 self.log.info("Saving file to %s", dest_key)
 
-                s3_hook.load_bytes(file_bytes, key=dest_key, replace=self.replace)
+                s3_hook.load_bytes(
+                    file_bytes, key=dest_key, replace=self.replace, s3_acl_policy=self.s3_acl_policy
```

Comment:
```suggestion
                    file_bytes, key=dest_key, replace=self.replace, acl_policy=self. s3_acl_policy
```
The `load_bytes` method has not `s3_acl_policy` argument.